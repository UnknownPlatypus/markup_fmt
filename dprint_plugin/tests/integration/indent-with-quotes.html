<script>
"{"; // This line cause panic
`
  `;
</script>

<script>
`{
}`;
``;
</script>

<script>
{
  `First line
   second line`;
}
</script>
