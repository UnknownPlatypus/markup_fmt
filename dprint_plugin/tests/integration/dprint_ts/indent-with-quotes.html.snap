---
source: dprint_plugin/tests/integration.rs
---
<script>
  "{"; // This line cause panic
  `
  `;
</script>

<script>
  `{
}`;
  ``;
</script>

<script>
  {
    `First line
   second line`;
  }
</script>
